opentelemetry-sdk/Cargo.toml

```diff
@@ -54,6 +56,7 @@ testing = ["opentelemetry/testing", "trace", "metrics", "logs", "rt-async-std",
 rt-tokio = ["tokio", "tokio-stream"]
 rt-tokio-current-thread = ["tokio", "tokio-stream"]
 rt-async-std = ["async-std"]
+use_hashbrown = ["hashbrown", "ahash"]
```

Comment:
`ahash` and `hashbrown` have been included as an _optional_ dependencies. Should the user opt to incorporate these dependencies, the HashMap and hashing functions supplied by these crates will be utilized. By default, the standard HashMap is used. Both crates seem to have wide customer adoption (based on the number of downloads). But if there are any concerns, these can be removed. The performance o

Reply:
given it is possible that the input (keys/values) can come from external sources, the default may be more safe against collision attacks, so lets add a warning/guide to advice endu sers on not relying on use_hashbrown flag, unless they control the keys/values.

Reply:
Have added this comment in Cargo.toml, purposefully didn't add the WARNING word :)

```
# Enable use_hashbrown for critical performance when measurement attributes are secure and not prone to external manipulation.
```